lib/matplotlib/_mathtext.py

```diff
@@ -2595,3 +2599,22 @@ def auto_delim(self, s, loc, toks):
             # if "mid" in toks ... can be removed when requiring pyparsing 3.
             toks["mid"].asList() if "mid" in toks else [],
             toks["right"])
+
+    def boldsymbol(self, s, loc, toks):
+        self.push_state()
+        state = self.get_state()
+        state.font = 'bfit'
+        hlist = []
+        name = toks["value"]
+        for c in name:
+            if isinstance(c, Char):
+                c.font = 'bfit'
+                c._update_metrics()
+                hlist.append(c)
+            elif isinstance(c, str):
+                hlist.append(Char(c, state))
```

Comment:
Can you think of a situation where this and the if-statement below is required? And in that case add it to the test?